src/samples/GroupChannelTypingIndicatorSample.js

```diff
@@ -0,0 +1,647 @@
+            fileMessageParams.file = e.currentTarget.files[0];
+            currentlyJoinedChannel.sendFileMessage(fileMessageParams)
+                .onSucceeded((message) => {
+                    const updatedMessages = [...messages, message];
+                    updateState({ ...state, messages: updatedMessages, messageInputValue: "", file: null });
+
+                })
+                .onFailed((error) => {
+                    console.log(error)
+                    console.log("failed")
+                });
+
+        }
+    }
+
+    const handleDeleteMessage = async (messageToDelete) => {
+        const { currentlyJoinedChannel, messages } = state;
+        await deleteMessage(currentlyJoinedChannel, messageToDelete);
+        const updatedMessages = messages.filter((message) => {
+            return message.messageId !== messageToDelete.messageId;
+        });
+        updateState({ ...state, messages: updatedMessages });
+
+    }
+
+    const updateMessage = async (message) => {
+        updateState({ ...state, messageToUpdate: message, messageInputValue: message.message });
+    }
+
+    const handleLoadMemberSelectionList = async () => {
+        updateState({ ...state, currentlyJoinedChannel: null });
+        const [users, error] = await getAllApplicationUsers();
+        if (error) {
+            return onError(error);
+        }
+        updateState({ ...state, currentlyJoinedChannel: null, applicationUsers: users, groupChannelMembers: [sb.currentUser.userId] });
+    }
+
+    const addToChannelMembersList = (userId) => {
+        const groupChannelMembers = [...state.groupChannelMembers, userId];
+        updateState({ ...state, groupChannelMembers: groupChannelMembers });
+
+    }
+
+    const setupUser = async () => {
+        const { userNameInputValue, userIdInputValue } = state;
+        const sendbirdChat = await SendbirdChat.init({
+            appId: SENDBIRD_INFO.appId,
+            localCacheEnabled: false,
+            modules: [new GroupChannelModule()]
+        });
+
+
+        await sendbirdChat.connect(userIdInputValue);
+        await sendbirdChat.setChannelInvitationPreference(true);
+
+        const userUpdateParams = new UserUpdateParams();
+        userUpdateParams.nickname = userNameInputValue;
+        userUpdateParams.userId = userIdInputValue;
+        await sendbirdChat.updateCurrentUserInfo(userUpdateParams);
+
+        sb = sendbirdChat;
+        updateState({ ...state, loading: true });
+        const [channels, error] = await loadChannels();
+        if (error) {
+            return onError(error);
+        }
+        console.log(sb.currentUser.userId);
+        updateState({ ...state, channels: channels, loading: false, settingUpUser: false });
+    }
+
+    if (state.loading) {
+        return <div>Loading...</div>
+    }
+
+    if (state.error) {
+        return <div className="error">{state.error} check console for more information.</div>
+    }
+
+    const displayTypingIndicator = () => {
```

Comment:
Could we please move this component down to to where other components are defined. starting line 334. Nice to keep that pattern going so users know where to look

Reply:
Also capital for react components please. DisplayTypingIndicator. 🙏🏻